docs/components/dropdown-menu.md

```diff
@@ -0,0 +1,394 @@
+# DropdownMenu
+
+**KDropdownMenu** is a button (or any slotted content) that is clicked to trigger a menu popover beneath it.
+
+<KDropdownMenu label="Documentation" :items="deepClone(defaultItemsUnselected)" />
+
+## Props
+
+### label
+
+The label for the menu.
+
+### items
+
+An array of items containing a `label` and other optional properties will render a menu of [`KDropdownItems`](#KDropdownItem) .
+
+```html
+<KDropdownMenu
+  label="Documentation"
+  :items="[
+    { label: 'Home', to: { path: '/' } },
+    { label: 'Button docs', to: { path: '/components/button.html' } },
+    { label: 'My docs', to: { path: '/components/dropdown-menu.html' } }
+  ]"
+/>
+```
+
+### appearance
+
+Use this prop to specify the display style for the dropdown menu. Can be either `menu` (default) or `selectionMenu`.
+The `menu` style is the standard you have seen in the example above.
```

Comment:
Can you mention here that it renders a KButton style, etc?

Reply:
Done